src/AgentHeaderDescriptor.php

```diff
@@ -40,6 +40,7 @@
 class AgentHeaderDescriptor
 {
     const AGENT_HEADER_KEY = 'x-goog-api-client';
+    const GAX_VERSION = '0.5.0-dev';
```

Comment:
Added. Unfortunately there isn't an easy way to do this for PHP, because the version number is defined in the git tag and nowhere else. But we can make a release process that bumps this version